tool/gravity/cli/rpcagent.go

```diff
@@ -348,6 +344,10 @@ func rpcAgentShutdown(env *localenv.LocalEnvironment) error {
 	return trace.Wrap(err)
 }
 
+func executeSyncOperationPlan(ctx context.Context, localEnv, updateEnv *localenv.LocalEnvironment, args []string) error {
```

Comment:
Since I'm reusing some of the update operation infrastructure and behavior for updating the environment variables, 
instead of `SyncOperationPlanToCluster`, agents are now started to sync the operation plan as the first thing to do.
except on the initiator node that creates the plan and runs the update operation. This is also relevant for update envars operation.